public/less/_admin.less

```diff
@@ -45,12 +45,12 @@
 		margin: 0;
 
 		dd {
-			margin-left: 240px;
+			margin-left: 275px;
```

Comment:
Why was this change necessary? Got any screenshots of how it looks before and after?